reconcile/utils/openshift_resource.py

```diff
@@ -498,6 +502,14 @@ def calculate_sha256sum(body):
         return m.hexdigest()
 
 
+def full_qualified_kind(kind: str, api_version: str) -> str:
```

Comment:
This is probably `fully_qualified_kind`?

Reply:
[2bbbf5e](https://github.com/app-sre/qontract-reconcile/pull/2356/commits/2bbbf5e022941d5ab7e6e03eb8f9751aa36ee8c2)